### DISCUSSION TITLE
22. La taille des images

### DISCUSSION THREAD

**Discussion by dNicolle** [cnumr/best-practices-wordpress#34] [Bonnes pratiques] [answered]
https://github.com/cnumr/fiches-wordpress-greenit/blob/main/best-practices-WordPress/fiches/22.%20La%20taille%20des%20images.md

**Comment by dNicolle** [comment]
@hrenaud il semble manquer GreenIT vous conseille et le principe de validation :)

**Reply by hrenaud** [reply]
C'est fait

**Comment by florinesueur** [comment, marked as answer]
Issue : https://github.com/cnumr/fiches-wordpress-greenit/issues/87
